styles/components/badges/styles.less

```diff
@@ -0,0 +1,450 @@
+& when (@badge-enabled) {
+  /* Badge */
+
+  .badge {
+    align-items: center;
+    display: inline-flex;
+    justify-content: center;
+    .element-shape-style(badge);
+    .element-spacing(badge, null, null);
+    .element-text-size(badge, null);
+    .element-text-style(badge);
+    .property-variant(badge, border-radius, null, null);
+
+    /* Layout Variants */
+
+    &.badge-narrow {
+      .element-spacing(badge, narrow, null);
+    }
+
+    &.badge-wide {
+      .element-spacing(badge, wide, null);
+    }
+
+    & > * {
+      display: inline-flex !important;
```

Comment:
@jfurrow happy to remove the `!important`, but worry about conflicts with elements being placed inside a badge that require `display: inline-flex;` to render properly.  As for the `*` selector, open to other ideas here, but essentially what I want is for any immediate child of the badge to observe inline-flex properties.  Suggestions?